### PR TITLE
Us 55

### LINKED ISSUE #28: User Story 55, Admin User Index Page
```
As an admin user
When I click a new "Users" link in the nav (only visible to admins)
Then my current URI route is "/admin/users"
Only admin users can reach this path.
I see all users in the system who are not merchants nor admins.
Each user's name is a link to a show page for that user ("/admin/users/5")
Next to each user's name is the date they registered
Next to each user's name is a button that says 'Upgrade to Merchant'
```

DO THESE STORIES FIRST:
- User Story 5, Admin Navigation

### Checklist:

- [x] base tests are written
- [x] base code is written to pass tests
- [x] edge case ("sad path") tests written (if required)
- [x] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #208

child of #274

depends on #268

### PR DESCRIPTION
**Pull request by WHomer**
What does this PR do? 
resolves #28.

The Promote to Merchant button is temporary set to root_path until the next user story.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -25,12 +25,17 @@
...
+              <%= link_to 'Users', admin_users_path, class: "nav-link" %>
+            </li>
+          <% end %>
         </ul>
         <ul class="navbar-nav mx">
           <% if current_user.nil? || !current_merchant? && !current_admin? %>
             <li class="nav-item active">
               <span class="nav-link" >
-                <%= link_to 'Cart', carts_path %>
+                <%= link_to 'Cart', carts_path, style:"text-decoration: none;" %>
```
Comment (VinceCarollo) [inline app/views/layouts/application.html.erb:38]: Now the link will match the others. Good fix. 👍

File: spec/features/navigation/application_spec.rb [hunk 2]
```diff
@@ -63,6 +63,7 @@
         within '.navbar' do
           expect(page).to have_no_link('Logout')
           expect(page).to have_no_link('Profile')
+          expect(page).to_not have_link('Users')
         end
```
Comment (VinceCarollo) [inline spec/features/navigation/application_spec.rb:67]: Good idea to add to old tests just in case.

**Review by VinceCarollo — APPROVED** [review]
